tests/unit/amo/components/TestBadge.js

```diff
@@ -1,45 +1,53 @@
-import { shallow } from 'enzyme';
 import * as React from 'react';
 
 import Badge from 'amo/components/Badge';
-import Icon from 'amo/components/Icon';
+import { render, screen } from 'tests/unit/helpers';
 
 describe(__filename, () => {
   it('renders a badge', () => {
-    const badge = shallow(<Badge label="super badge" />);
+    const label = 'badge label';
+    const { root } = render(<Badge label={label} />);
 
-    expect(badge).toHaveClassName('Badge');
-    expect(badge.find(Icon)).toHaveLength(0);
-    expect(badge.children()).toIncludeText('super badge');
+    expect(root).toHaveClass('Badge');
+    expect(screen.queryByClassName('Icon')).toHaveLength(0);
+    expect(screen.getByText(label)).toBeInTheDocument();
   });
 
   it('can override the icon label', () => {
-    const badge = shallow(<Badge type="experimental" label="foo" />);
+    const label = 'foo';
+    render(<Badge type="experimental" label={label} />);
 
-    expect(badge.find(Icon)).toHaveProp('alt', 'foo');
```

Comment:
Does this render as an alt property?  if so, does `ByAltText` work here?

Reply:
I think there may be some places we can use `ByAltText` in this patch, but this is not one of them. It only works with actual `alt` attributes: https://testing-library.com/docs/queries/byalttext, so it doesn't find anything in this component.